3-ecommerce-db/projects/cyf-ecommerce-api/server/server.js

```diff
@@ -40,7 +40,7 @@ app.get("/suppliers", async (req, res) => {
 
 
 
-app.listen(3000, function () {
+app.listen(3001, function () {

```

Comment:
normally, there is no need in changing this port number.
if you couldn't start it that means you had another application instance already running and keeping port 3000 busy. usually you just need to check other console terminals and see if any node app is running there.

however, you might intentionally want to run several apps simultaneously - then in that case you should make sure they all use different ports.

and one more note here: 
it's common practice in Full Stack Development to have port 3000 reserved for a front-end and port 4000 or 5000 for back-end. here we are dealing with the back-end.